src/refactorings/convert-switch-to-if-else/convert-switch-to-if-else.ts

```diff
@@ -24,7 +24,11 @@ function updateCode(ast: t.AST, selection: Selection): t.Transformed {
   return t.transformAST(
     ast,
     createVisitor(selection, (path, convertedNode) => {
-      path.replaceWith(convertedNode);
+      if (t.isBlockStatement(convertedNode)) {
```

Comment:
Returning a block node to convey that there are multiple nodes to replace may not be the cleanest solution but it worked

Reply:
That's fine, for the moment it works. We can still refactor the code later if we need to change it, thanks to the tests you've written it's easy to do so 👍